src/components/ChatItem.tsx

```diff
@@ -0,0 +1,45 @@
+import {useRecoilValue } from 'recoil';
+import {MyMessageWrapper,MyMessage,MySendTime, PartnerMessage,UserImage,MessageWrapper,UserName,UserInfoWrapper,UserMessage,SendTime } from '../styles/style.chatitem';
+import {userInfo,partnerInfo} from '../store/atom';
+import {IChatProps,IUser} from '../store/interface';
+
+function ChatItem({id,userid,message,time,user} : IChatProps) : JSX.Element {
+  const currentUser = useRecoilValue<IUser>(userInfo);
+  const partnerUser = useRecoilValue<IUser>(partnerInfo);
+  return (
+    <>
+      {userid === currentUser.userid ? (
+        <MyMessageWrapper>
+          <MySendTime>
+          {time}
+          </MySendTime>
+          <MyMessage>
+            {message}
+          </MyMessage>
+        </MyMessageWrapper>
+      ) : (
+        <PartnerMessage>
+          <UserImage 
+            src = {`${process.env.PUBLIC_URL}/images/${userid}.jpg`} 

```

Comment:
```suggestion
            src = {`/images/${userid}.jpg`} 
```
와 같이 public 내 img를 더 간편하게 불러올 수 있습니다!